charts/fleet/Chart.yaml

```diff
@@ -8,7 +8,7 @@ version: v6.4.2
 home: https://github.com/fleetdm/fleet
 sources:
   - https://github.com/fleetdm/fleet.git
-appVersion: v4.64.1
+appVersion: v4.65.0
```

Comment:
@georgekarrv Should the chart version be incrementing or not merged yet?

Reply:
Not merged yet. Still awaiting your approval

Reply:
https://github.com/fleetdm/fleet/pull/26535

Reply:
🤦 My bad.